lib/config.js

```diff
@@ -2,7 +2,6 @@ module.exports = {
   noAuthNeeded: [
     'login',
     'guide',
-    'snake',
```

Comment:
Nice, since this is a "breaking change", can we please have it as part of V2 against the `beta` branch?

Reply:
Have you seen my comment about it not being a breaking change because it wasn't part of the official api? Do you think it will actually break someones setup?

Reply:
Ok you convinced me I'll change merge base to beta. Don't want you to need to go through beta rebase again.